bitnami/postgresql-ha/Chart.yaml

```diff
@@ -29,4 +29,4 @@ name: postgresql-ha
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/postgresql
   - https://www.postgresql.org/
-version: 11.2.0
+version: 11.2.1
```

Comment:
With the default settings this change is not affecting the default deployment, but I would increase the version to `11.3.0` for this change.

Reply:
Fixed. Sorry for a delay